iFixit/react-commerce
# Upgrade to React Query v4

closes #995 

## QA

Perform a smoke test to verify the the app still works. The features affected by this change are:
- auth
- cart
- product list related posts section
- product page reviews
